how-to-guides/hello-neptune/notebooks/hello_neptune.ipynb

```diff
@@ -151,9 +151,9 @@
     "    capture_hardware_metrics=True,\n",
     "    capture_stderr=True,\n",
     "    capture_stdout=True,\n",
-    "    source_files=\"hello_neptune.ipynb\",  # does not work in Colab\n",
+    "    source_files=\"hello_neptune.ipynb\",  # Doesn't work in Colab\n",
     "    tags=[\"quickstart\", \"notebook\"],\n",
-    "    dependencies=\"requirements.txt\",  # pass the path to the requirements.txt file\n",
+    "    dependencies=\"requirements.txt\",  # You can pass the path to your dependencies file or omit this parameter\n",
```

Comment:
Dunno when this behaviour changed, but it looks like `infer` now works for notebooks too!
cc: @normandy7 

```suggestion
    "    dependencies=\"infer\",  # You can also pass the path to your dependencies file or omit this parameter\n",
```